src/server.js

```diff
@@ -100,6 +113,14 @@ in ${config.public ? "public" : "private"} mode`);
 
 		manager.autoloadUsers();
 	}
+
+	function shutdown() {
+		log.info("Quitting...");
+		server.close(); // Removes socket file
+		process.exit();
+	}
+
+	process.on("SIGINT", shutdown);
```

Comment:
It seems to be necessary in order to get rid of the socket file when the server shuts down. Otherwise, the socket binding fails next time when Lounge starts since the file already exists. Do you know any better way to achieve this?